P3481.bs

```diff
@@ -185,27 +256,31 @@ Passing a range as parameters to the functor passed to `bulk_chunked` is similar
 An implementation of `bulk` can be easily built on top of `bulk_chunked` without losing performance (as shown in the Proposal section).
 
 
-## Using execution policies ## {#policies}
+## Also define an unchunked version ## {#design.unchunked}
 
-As discussed in the Motivation section, not having the possibility to specify execution policies for the `bulk` algorithm is a downside.
-If the default execution policy for regular algorithms is seq, for `bulk` (and `bulk_chunked`) it makes more sense for the default to be par or par_unseq.
-But, there are cases in which any of these two would be more appropriate.
-Thus, we cannot choose a default policy.
+For GPUs is important to have a version of `bulk` that ensures one execution agent per iteration.
+Defining `bulk` in terms of `bulk_chunked` would not be efficient for GPUs.
 
-One criticism of this solution is that each invocation of `bulk` needs to contain an extra parameter that is typically verbose to type.
-But there is always an easy workaround: the user can define a thin wrapper on top of `bulk` / `bulk_chunked` that calls the algorithm with the default execution policy.
-Something like:
+Moreover, for composability reasons, we need a way to specify which version of `bulk` will not do any chunking.
```

Comment:
Please elaborate what composability you are talking about and also composability between what and what...

Reply:
Added example.